src/main.rs

```diff
@@ -127,25 +144,40 @@ impl Args {
 
 #[derive(Copy, Clone, Debug)]
 struct Limits {
+    /// Size in megabytes
     size_limit_default: usize,
```

Comment:
I think these should be bytes. We should (eventually) provide functionality to be able to specify e.g. `10m` as a valid config value

Reply:
It used to be in bytes a few weeks ago but was changed to megabytes outside of this pr. This just documents what these fields already do. Also I can't really see a use case for taking kilobytes or gigabytes as an argument